src/components/AccountPickerItem.jsx

```diff
@@ -36,37 +38,49 @@ export const AccountPickerItem = ({
         <span>{accountName}</span>
         {nameAndLoginDiffer && <small>{accountLogin}</small>}
       </div>
-      {hasError && (
-        <div
-          className={classNames(
-            styles['col-account-picker-button-status'],
-            styles['col-account-picker-button-status--error']
-          )}
-        >
-          <span>
-            {getErrorTitle(t, error, key => `connection.error.${key}.title`)}
-          </span>
-          <Icon icon="warning" />
+      {maintenance && (
+        <div className={styles['col-account-picker-button-status']}>
+          <span>{t('connector.maintenance')}</span>
         </div>
       )}
-      {hasUpdate &&
+      {!maintenance &&
+        hasError && (
+          <div
+            className={classNames(
+              styles['col-account-picker-button-status'],
+              styles['col-account-picker-button-status--error']
+            )}
+          >
+            <span>
+              {getErrorTitle(t, error, key => `connection.error.${key}.title`)}
+            </span>
+            <Icon icon="warning" />
+          </div>
+        )}
+      {!maintenance &&
+        hasUpdate &&
         !hasError && (
           <div className={styles['col-account-picker-button-status']}>
             <span>{t('connector.update')}</span>
           </div>
         )}
-      {isConnected &&
+      {!maintenance &&
+        isConnected &&
         !hasUpdate && <Icon icon="check-circleless" color="#2BBA40" />}
-      {isRunning && <Spinner />}
+      {!maintenance && isRunning && <Spinner />}
     </NavLink>
   )
 }
 
-const mapStateToProps = (state, ownProps) => ({
-  account: getAccount(
-    state.cozy,
-    ownProps.connection && ownProps.connection.account
-  )
-})
+const mapStateToProps = (state, ownProps) => {
+  const { konnectorSlug } = ownProps
+  return {
+    account: getAccount(
+      state.cozy,
+      ownProps.connection && ownProps.connection.account
+    ),
+    maintenance: getKonnectorsInMaintenance()[konnectorSlug]
```

Comment:
Agree with that, we will have to replace also in the other parts using maintenance informations.
But this will disappear with the maintenance from the registry at term.